prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <a href="http://localhost:5500/prep-html-links/index.html">Click here to go to the home page</a>
```

Comment:
you don't want to use localhost here, just the relative path from the file you are in to the file you want to access.